lib/generators/draft/resource/templates/views/show.html.erb

```diff
@@ -15,8 +15,8 @@
 <% if with_sentinels? -%>
       <!-- Delete link <%= singular_table_name %> start -->
 <% end -%>
-      <div class="col">
-        <a href="/delete_<%= singular_table_name %>/<%%= @<%= singular_table_name %>.id %>" class="btn btn-block btn-outline-secondary">
+      <div>
+        <a href="/delete_<%= singular_table_name %>/<%%= @<%= singular_table_name %>.id %>">
```

Comment:
@pmckernin 🤔 I'm not sure how the update from https://github.com/firstdraft/draft_generators/blob/1b26414951e51b4e1d45307eea5bc2745b4fb594/lib/generators/draft/resource/templates/views/show.html.erb#L31 didn't make it into this branch, but it didn't. See #97